R/model_info.R

```diff
@@ -78,7 +84,10 @@ model_info.svm <- function(model, ...) {
 model_info.glm <- function(model, ...) {
   type <- "regression"
   package <- "stats"
-  ver <- as.character(utils::packageVersion("stats"))
+  ver <- try(as.character(utils::packageVersion(package)), silent = TRUE)
```

Comment:
these 4 lines are repeated multiple times
according to DRY (do not repeat yourself) this shall be converted into a small function,
e.g. safe_check_pkg_version()

Reply:
Done